### ISSUE TITLE
update frontend for delete feature

### ISSUE THREAD

**Issue by Thykof** [massalabs/thyra-plugin-wallet#311]
after #310 is merged, update the frontend setting page to redirect to the right page:
- stay in setting page in case of failure
- navigate to account selection page (/index) in case of success.

**Comment by Thykof** [comment]
@SlnPons I wrote this issue, can you validate the user flow?

- User is in the setting page
- click on "delete account"
- in case of deletion sucess
  - navigate to account selection page (/index) in case of success.
- in case of detetion failure or canceled 
  - stay on the settings page

**Comment by SlnPons** [comment]
@Thykof perfect! Well done :)

**Comment by Thykof** [comment]
fortunately already done!
